CHANGELOG.md

```diff
@@ -96,6 +96,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Add community_id ingest processor ([#12121](https://github.com/opensearch-project/OpenSearch/pull/12121))
 - Introduce query level setting `index.query.max_nested_depth` limiting nested queries ([#3268](https://github.com/opensearch-project/OpenSearch/issues/3268)
 - Add toString methods to MultiSearchRequest, MultiGetRequest and CreateIndexRequest ([#12163](https://github.com/opensearch-project/OpenSearch/pull/12163))
+- Views ([#11957](https://github.com/opensearch-project/OpenSearch/pull/11957))
```

Comment:
Should we add a more descriptive changelog entry?

Reply:
Yes, please! And make sure the squashed commit has a good message.

@peternied Also it is better to move this to the `[Unreleased 3.0]` until we have a plan to backport it to 2.x.